LucidGloves/src/DeviceDriver/LucidGloveDriver.cpp

```diff
@@ -57,7 +57,7 @@ vr::EVRInitError LucidGloveDeviceDriver::Activate(uint32_t unObjectId) {
 	const bool isRightHand = IsRightHand();
 
 	m_driverId = unObjectId; //unique ID for your driver
-	m_controllerPose = std::make_unique<ControllerPose>(m_configuration.role, std::string(lucidGlove::c_deviceManufacturer), m_configuration.offsetVector, m_configuration.angleOffsetVector, m_configuration.controllerIdOverride, m_driverId);
+	m_controllerPose = std::make_unique<ControllerPose>(m_configuration.role, std::string(lucidGlove::c_deviceManufacturer), m_configuration.offsetVector, m_configuration.angleOffsetVector, m_configuration.controllerIdOverride, m_configuration.isControllerOverride, m_driverId);
```

Comment:
perhaps we can just provide the configuration to ControllerPose as well by casting to a shared pointer

We probably need to have a rethink to how we pass configuration, seems a little messy (though thats my fault)

Reply:
Yeah, let's do that in a separate PR